### DISCUSSION TITLE
Make Ruby on Solaris 10 with ripper.so and rb_mjit_min_header-2.7.5.h errors

### DISCUSSION THREAD

**Discussion by ghost** [rbenv/ruby-build#1893] [Build failures]
make[1]: Entering directory '/export0/home/telegraf/fluentd/ruby_versions/ruby275/ruby-2.7.5'
make[1]: Nothing to be done for 'encs'.
make[1]: Leaving directory '/export0/home/telegraf/fluentd/ruby_versions/ruby275/ruby-2.7.5'
building rb_mjit_header.h
rb_mjit_header.h unchanged
building .ext/include/i386-solaris2.10/rb_mjit_min_header-2.7.5.h
./miniruby -I./lib -I. -I.ext/common  ./tool/transform_mjit_header.rb "/usr/sfw/bin/gcc  -std=c99 " rb_mjit_header.h .ext/include/i386-solaris2.10/rb_mjit_min_header-2.7.5.h
error in initial header file:
cc1: error: unrecognized command line option "-Werror=implicit-function-declaration"
cc1: error: unrecognized command line option "-Werror=implicit-int"
cc1: error: unrecognized command line option "-Wfatal-errors"
make: [Makefile:928: .ext/include/i386-solaris2.10/rb_mjit_min_header-2.7.5.h] Error 1 (ignored)
ln -sf ../../../.ext/include/i386-solaris2.10/rb_mjit_min_header-2.7.5.h include/ruby-2.7.0/i386-solaris2.10/rb_mjit_min_header-2.7.5.h
Generating RDoc documentation
uh-oh! RDoc had a problem:
ld.so.1: ruby: fatal: Init_ripper: can't find symbol - /export0/home/telegraf/fluentd/ruby_versions/ruby275/ruby-2.7.5/.ext/i386-solaris2.10/ripper.so

run with --debug for full backtrace
make: [Makefile:1243: rdoc] Error 1 (ignored)
# 

======

# /opt/csw/gnu/make -v
GNU Make 4.3
Built for i386-pc-solaris2.10

**Comment by hsbt** [comment]
@dmitriklein Again, It's not ruby-build issue. Unfortunately, I will block you because you report the irrelevant report for this project.

**Comment by hsbt** [comment]
@dmitriklein Basically, you should ignore the multi-posting your issues to GitHub repos, Ruby bug tracker as https://bugs.ruby-lang.org/issues/18429
